### ISSUE TITLE
`pyright --verifytypes` does not consider funcs with one unannotated argument to have unknown type

### ISSUE THREAD

**Issue by smackesey** [microsoft/pyright#3218]
`pyright --verifytypes` appears to think that unannotated functions with a single parameter have known type:

```
# foo/__init__.py in a py.typed package `foo`
def foo(bar) -> str:
    return bar or ''

$ pyright --verifytypes foo
...
Symbols exported by "fakepkg": 1
  With known type: 1
  With ambiguous type: 0
  With unknown type: 0
  Functions without docstring: 1
  Functions without default param: 0
  Classes without docstring: 0
```

Notice that `foo` is considered to have a "known" type despite the absence of annotation on `bar` param. But add a second parameter, and now it has an unknown type:

```
# foo/__init__.py in a py.typed package
def foo(bar, baz) -> str:
    return bar or ''

$ pyright --verifytypes foo
...
Symbols exported by "fakepkg": 1
  With known type: 0
  With ambiguous type: 0
  With unknown type: 1
  Functions without docstring: 1
  Functions without default param: 0
  Classes without docstring: 0
```

**VS Code extension or command-line**

`pyright` CLI, v1.1.230

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release. There was a bug where the logic was treating all functions as though they were instance methods, so it assumed that the first parameter was a "self" parameter, and it allowed them to go unannotated.

**Comment by erictraut** [comment]
This is fixed in pyright 1.1.231, which I just published.
